install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   namespace: openshift-machine-config-operator
   labels:
     k8s-app: machine-config-daemon
+  annotations:
```

Comment:
I think we want these Prometheus rules even in external control plane scenarios, right?

Reply:
ack, will remove this change